server/src/main/java/org/apache/uniffle/server/buffer/AbstractShuffleBuffer.java

```diff
@@ -41,8 +41,12 @@ public abstract class AbstractShuffleBuffer implements ShuffleBuffer {
 
   protected long size;
 
+  protected volatile boolean closed;
```

Comment:
the naming of closed sounds weired to me. How about dropped? or evicted?

Reply:
> the naming of closed sounds weired to me. How about dropped? or evicted?

How about "released"? More Closer to the actual meaning.

Reply:
@zuston